.npmrc

```diff
@@ -0,0 +1,2 @@
+@leancode:registry=https://f.feedz.io/leancode/public/npm
+legacy-peer-deps=true
```

Comment:
Not really, though we will have to downgrade some packages like `@typescript-eslint/eslint-plugin` and `eslint-plugin-unused-imports` so that they don't conflict with older versions specified in `@leancode/eslint-config`

Reply:
👍